packages/shared/src/components/HorizontalScroll/HorizontalScroll.spec.tsx

```diff
@@ -31,8 +44,7 @@ describe('HorizontalScroll', () => {
     const mockOnClickSeeAll = jest.fn();
     render(
       <HorizontalScroll
-        title="Scrollable Area"
-        onClickSeeAll={mockOnClickSeeAll}
+        scrollProps={{ onClickSeeAll: mockOnClickSeeAll, title: 'Scrollable ' }}
```

Comment:
Why is it `Scrollable` here and `Scrollable Area` elswhere?

Reply:
Just tests, nothing concerning, we can change the name here anyways if we want to have it uniform